src/pages/index.js

```diff
@@ -44,6 +52,17 @@ const IndexPage = () => {
         imageSource={images.partnerships.childImageSharp.fluid}
         altText="Explore The Partnerships"
       />
+      <StoryBlock
+        imageSource={images.barriers.childImageSharp.fluid}
+        altText="Barriers"
+        description="Hellow word!"
+      />
+      <StoryBlock
+        imageSource={images.barriers.childImageSharp.fluid}
+        altText="Barriers"
+        description="Hellow word!"
+        imageRight
```

Comment:
see here, since the prop is a boolean value (true or false), but just including it we mean 'true' (imageRight=true)